MarkupBatchConversionHandler.inc.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * @file plugins/generic/markup/MarkupBatchConversionHandler.inc.php
+ *
+ * Copyright (c) 2003-2016 Simon Fraser University Library

```

Comment:
All dates -2017

Reply:
Also, all copyrights to Simon Fraser University, not Simon Fraser University library.